engine/src/lib.rs

```diff
@@ -295,30 +325,85 @@ mod contract {
         io.return_output(&data[..]);
     }
 
+    /// Starts the hashchain from indicated block height and block hashchain values.
+    /// Resumes the contract.
+    /// Requires a specific Aurora Labs account.
+    /// Requires contract to be in pause state.
+    /// Requires that the indicated block height is before the current block height.
+    /// Assumes that no tx has been accepted after the last tx included on the indicated block hashchain.
+    /// This self tx is added to the started hashchain as it changes the state of the contract.
+    #[no_mangle]
+    #[named]
+    pub extern "C" fn start_hashchain() {
+        let mut io = Runtime;
+        let mut state = state::get_state(&io).sdk_unwrap();
+
+        // *** TODO requires some Aurora Labs account
+        // require_account(some_AuroraLabs_account);
```

Comment:
Please add a `require_account` statement here for an account owned by Aurora Labs. For security reasons, it's better that I don't create the account.